### DISCUSSION TITLE
Contributions not appears..

### DISCUSSION THREAD

**Discussion by Phil-dev8** [community/community#55216] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello devs
Since the 13 march, my contributions not appears on my profile...  I didn't touch any settings in my profile, my mail is valid..
I checked this link: https://docs.github.com/fr/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile
and I didn't understand what go wrong....
Can you help me please?
Thanks

**Comment by Phil-dev8** [comment]
Hey, anyone to help me, I don't understand what happen

**Comment by airtower-luna** [comment, marked as answer]
Looking at your two most recently updated repositories, none of the commits are recognized as yours. In the most recent commits ([example](https://github.com/Phil-dev8/deliveroo-backend/commit/e499b30634411db0bce82f42f4d554d4a3199e15)) the author name seems to be set to a mail address, while the author email address is invalid (`.home` is currently not a TLD).

So unfortunately your `user.email` setting is *not* valid, you need to set it to an email address connected to your account. I'd recommend also fixing `user.name`. Mind that this only affect *new* commits, if you want to fix past ones you have to rewrite the repository history, or add a mailmap file.

**Reply by eivindjahren** [reply]
Does github respect the `.mailmap` file for https://github.com/org/repo/commits?author=AUTHOR and https://github.com/org/repo/graphs/contributors ? As far as I can tell, it does not.

**Comment by admont02** [comment]
Try this:

Open the command line.
Set your username: git config --global user.name "FIRST_NAME LAST_NAME"
Set your email address: git config --global user.email "MY_NAME@example.com"

**Comment by chrome-dome** [comment]
I recently had to refresh my system. When creating new projects, of course, I linked everything (VSC) to the GitHub account. But it slipped my mind that I still need to do it in the terminal. if you work as usual for a long time, you forget what needs to be done at the beginning :). The last two months of my work have not been saved in the contribution section. It was enough to set the username and e-mail address in the terminal.
Like @admont02 wrote, on the command line after entering the above instructions, if you don't get any feedback, it means it's not set. And then you set it as mentioned, where from the user name and email address you enter your name from the git hub settings(public profile section)
